docroot/themes/humsci/humsci_basic/templates/components/logo.html.twig

```diff
@@ -0,0 +1,16 @@
+{#
+  /**
+  * @file
+  * Logo Component
+  *
+  * The Stanford wordmark logo rendered with the custom Stanford ligature font.
+  *
+  * Available variables:
+  * - attributes: For additional HTML attributes not already provided.
+  * - modifier_class: Additional CSS classes to change look and behaviour.
+  * - href: The URL that the logo links to.
+  * - logo_text: The text for the ligature logo - either Stanford<br>University or Stanford.
+  */
+#}
+
+<a {{ attributes }} class="su-logo {{ modifier_class }}" href="{{ href|default('https://www.stanford.edu') }}">{{ logo_text|default('Stanford<br>University')|raw }}</a>
```

Comment:
can you nest the text and then drop the closing tag to the next line as well? It'll be a bit more readable that way :)

Reply:
Oh yes! That's much better. :smile: